geode-core/src/main/java/org/apache/geode/internal/cache/map/RegionMapDestroy.java

```diff
@@ -528,6 +528,9 @@ private void handleEntryNotFound() {
     boolean throwException = false;
     EntryNotFoundException entryNotFoundException = null;
 
+    if(isEviction)
```

Comment:
This is just a code style thing but add curlys around the return statement.

Reply:
done!